### ISSUE TITLE
do you know how to train crosswoz dataset  in tutorial

### ISSUE THREAD

**Issue by weizhenzhao** [ConvLab/ConvLab-3#120]
`from convlab.base_models.t5.nlu import T5NLU
from convlab.base_models.t5.dst import T5DST
from convlab.base_models.t5.nlg import T5NLG
from convlab.policy.vector.vector_nodes import VectorNodes
from convlab.policy.vtrace_DPT import VTRACE
from convlab.dialog_agent import PipelineAgent, BiSession
from pprint import pprint
import random
import numpy as np
import torch

# go to README.md of each model for more information
sys_nlu = T5NLU(speaker='user', context_window_size=0, model_name_or_path='ConvLab/mt5-small-nlu-all-crosswoz')
sys_dst = T5DST(dataset_name='crosswoz', speaker='user', context_window_size=100, model_name_or_path='ConvLab/mt5-small-dst-crosswoz')
# Download pre-trained DDPT model
vectorizer = VectorNodes(dataset_name='crosswoz',
                         use_masking=True,
                         manually_add_entity_names=True,
                         seed=0,
                         filter_state=True)
sys_policy = VTRACE(is_train=False,
              seed=0,
              vectorizer=vectorizer,
              load_path="convlab/policy/vtrace_DPT/supervised")
sys_nlg = T5NLG(speaker='system', context_window_size=0, model_name_or_path='ConvLab/mt5-small-nlg-all-crosswoz')
# assemble
sys_agent = PipelineAgent(sys_nlu, sys_dst, sys_policy, sys_nlg, name='sys')
sys_agent.init_session()
print(sys_agent.response("帮我推荐一个人均消费在50-100元的餐馆"))`
### Checkbox

I follow the instructions in the tutorial, and change the dataset to crosswoz.
but it always return the following result. 

Load actions from file..
Dimension of system actions: 89
Dimension of user actions: 81
State dimension: 232
Load actions from file..
Dimension of system actions: 89
Dimension of user actions: 81
State dimension: 232
Load actions from file..
Dimension of system actions: 89
Dimension of user actions: 81
State dimension: 232
WARNING:root:nlu info_dict is not initialized
WARNING:root:dst info_dict is not initialized
WARNING:root:nlg info_dict is not initialized
[['Inform', '餐馆', '人均消费', '50-100元'], ['Request', '餐馆', '名称', '']]
{'belief_state': {'餐馆': {'人均消费': '50-100元'}}, 'booked': {}, 'history': [['sys', 'null'], ['user', '帮我推荐一个人均消费在50-100元的餐馆']], 'system_action': [], 'user_action': [['Inform', '餐馆', '人均消费', '50-100元'], ['Request', '餐馆', '名称', '']], 'terminated': False}
[['NoOffer', '餐馆', 'none', 'none']]
抱歉,没有查到符合条件的餐馆信息,你看可以吗?
抱歉,没有查到符合条件的餐馆信息,你看可以吗?

Process finished with exit code 0

I think it's fail to run
If anyone can help .
thanks very much

**Comment by zqwerty** [comment]
See https://github.com/ConvLab/ConvLab-3/issues/116#issuecomment-1380365310

**Comment by ChrisGeishauser** [comment]
I updated to code to use crosswoz with DDPT, but i did not check properly if supervised training works. You can try it using

python convlab/policy/vtrace_DPT/supervised/train_supervised.py --dataset_name=crosswoz

**Comment by lowpair** [comment]
@weizhenzhao Can you run normally after the latest update?

**Comment by weizhenzhao** [comment]
<img width="2037" alt="image" src="https://user-images.githubusercontent.com/13706689/219946047-1997ca02-3b1e-4345-80fc-e94e8c9b44cf.png">
I've tried the change. but there's still something wrong here.
it see's that after preprocessing . the data in the action list is all zero.
and It went wrong in that line 
<img width="1739" alt="image" src="https://user-images.githubusercontent.com/13706689/219946135-94e68681-1fae-4ae8-a1e9-e0d7da35a9fd.png">

**Comment by ChrisGeishauser** [comment]
Hey @weizhenzhao ! Thanks a lot for pointing me towards that. The issue was that the intents in crosswoz were upper-case and then it couldn't match the lowercase one in the action dictionary. I changed one line (and pushed the change to master) and for me it now minimizes the loss. Please let me know if you face more issues then!

**Comment by dyy401453043** [comment]
<img width="709" alt="image" src="https://user-images.githubusercontent.com/39038834/228452995-3968ad0f-84c4-4ab8-834e-ac28f262e419.png">
face problem when i run the code in crosswoz like https://github.com/ConvLab/ConvLab-3/issues/120#issue-1537141658.
